src/OrleansRuntime/MembershipService/GrainBasedMembershipTable.cs

```diff
@@ -32,6 +33,17 @@ internal class GrainBasedMembershipTable : Grain, IMembershipTable
         private InMemoryMembershipTable table;
         private TraceLogger logger;
 
+        public Task InitializeMembershipTable(GlobalConfiguration config, bool tryInitTableVersion, TraceLogger traceLogger)
+        {
+            return OnActivateAsync();
```

Comment:
ok, i think InitializeMembershipTable and DeleteMembershipTableEntries should just be:
```csharp
throw new InvalidOperationException("This method should not be called directly");
```
because it's there only as stubs and breaks the grain flow.